README.md

```diff
@@ -1 +1,35 @@
-# sdcore-upf-snap
\ No newline at end of file
+# SD-Core User Plane Function (UPF) Snap
+
+A snap for the SD-Core User Plane Function
+
+## Usage
+
+### Install
+
+```bash
+sudo snap install sdcore-upf
+```
+
+### Configure
+
+Configure the upf bessd by editing the `/var/snap/sdcore-upf/common/upf.json` file.
```

Comment:
Should we mention some reference here or do we assume that the potential user knows what to do?
Some configuration basics: https://docs.sd-core.opennetworking.org/master/configuration/config_upf.html

Reply:
The README.md isn't user documentation, ideally users don't end up here. This being said, I'll add those reference to the snap information in snapcraft.io.